ansible/roles/stack-sunbird/templates/ml-survey-service.env

```diff
@@ -0,0 +1,151 @@
+# ML Survey Service Config
+# Application host
+HOST={{ml_survey_host | default("localhost")}}
+
+# Application port number which the service will listen to
+PORT={{ml_survey_port | default("3000")}} 
+
+#Application ENV example local/dev/production
+NODE_ENV={{ml_survey_node_env|default("production")}}
```

Comment:
Default should be dev, for production we should have a override in private.

Reply:
changed to development